scripts/verify-storage-persistence.sh

```
@@ -0,0 +1,874 @@
+        return 1
+    fi
+    
+    wait_for_api_healthy || return 1
+    
+    return 0
+}
+
+# Run test for a single storage mode
+run_single_mode_test() {
+    local mode="$1"
+    local test_name="test-${mode}-$(date +%s)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)